game.py

```diff
@@ -65,7 +65,13 @@ def __init__(self, bg_image_path, keymap, config,
                                   pane_height, 4])
 
         self.generator = Generator()
-        self.generator.configure(spikes_config)
+        if _type == "spikes":
+            self.generator.configure(spikes_config, "spike")
+        elif _type == "cardiac":
+            self.generator.configure(spikes_config, "heart")
+        else:
+            raise Exception("Incorrect game mode")
+
```

Comment:
Why this is pythonic, it's not consistent with Sugar activities. 
Instance variables are referred to by `self.name` as that's less ambiguous.

Reply:
Hmm missed it, Corrected it.